### ISSUE TITLE
ideas for the report

### ISSUE THREAD

**Issue by Kkkeren7** [BigCokee/G14_Encyclopedia#4]
Does part of the design process need to include why we chose the ThemeRivers

**Comment by YuhanMA1** [comment]
Yes, ThemeRivers is a specific visualisation method so we need to explain to readers what it is, and it is an important part of our visualisation.

**Comment by Kkkeren7** [comment]
My point
1.Why did we choose a ThemeRivers?
2.How were trade trended presented?
3.Overall design style

**Comment by Shiiiijie** [comment]
On the first minor point.
ThemeRiver can visually present trends in the number of subject terms for different topics, making it easy for us to find a topic that is changing more significantly and continue to delve into the reasons for its change, and for our readers to get a quick overview of the topics we have filtered out.
